### PR TITLE
Add timer

### PR DESCRIPTION
**Pull request by Faffie**
Added game timer to game session. 

https://github.com/alanvanrossum/kroket/pull/42

### CODE REVIEW

File: EscapeHost/src/main/java/nl/tudelft/kroket/server/GameSession.java [hunk 1]
```diff
@@ -203,4 +209,16 @@ public void handleMessage(String input, HashMap<String, String> parsedInput) {
     }
   }
 
+  public void update() {
+
+    long timeRemaining = Math.max(timeLimit - System.currentTimeMillis(), 0);
+
+    // TODO: stop the game if timeRemaining reaches value <= 0
+
+    if (timeRemaining <= 0) {
```
Comment (Faffie) [inline EscapeHost/src/main/java/nl/tudelft/kroket/server/GameSession.java]: Hm, this should only be fired once. Perhaps add a check.

File: EscapeHost/src/main/java/nl/tudelft/kroket/server/Settings.java [hunk 2]
```diff
@@ -25,4 +25,6 @@
 
   /** Number of threads available in the pool. */
   public static final int THREAD_POOL_MAX = 10;
+
+  public static final int TIMELIMIT = 23 * 60;
```
Comment (Faffie) [inline EscapeHost/src/main/java/nl/tudelft/kroket/server/Settings.java]: I used an odd value to test this, as the VR client has a different default value. This value will overrule the timer in the VR client.

**Comment by irenevanderblij** [comment]
The host never seems to send the gameover message to the clients when the time is up

**Comment by Faffie** [comment]
You're right, forgot to commit&push a change.

**Comment by irenevanderblij** [comment]
Gamewon and gamelost screens both work now. Please review and merge!

**Comment by alanvanrossum** [comment]
heel mooi

**Pull request merged**